docs/usage/administration/admin_api/README.md

```diff
@@ -0,0 +1,28 @@
+# The Admin API
+
+## Authenticate as a server admin
+
+Many of the API calls in the admin api will require an `access_token` for a
+server admin. (Note that a server admin is distinct from a room admin.)
+
+A user can be marked as a server admin by updating the database directly, e.g.:
+
+```sql
+UPDATE users SET admin = 1 WHERE name = '@foo:bar.com';
+```
+
+A new server admin user can also be created using the
+[register_new_matrix_user](../../../../scripts/register_new_matrix_user) script.
```

Comment:
This link doesn't seem to work?

Reply:
Ah yes, we can only direct link to things that will be included in the documentation. This script won't, obviously. This probably doesn't need to be a link ([the actual script itself is tiny](https://github.com/matrix-org/synapse/blob/develop/scripts/register_new_matrix_user)). I've called it a command instead and provided a bit more detail about where one may find it locally.